lexicons/com/atproto/server/getSigningKey.json

```diff
@@ -0,0 +1,23 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.server.getSigningKey",
```

Comment:
I think I'd prefer an indication in the method name that this is for account creation.

Even if we go the verbose route and say `server.getSigningKeyForNewAccount` or  similar. Otherwise I feel like this looks like it just returns _the server's_ signing key